assets/src/edit-story/panels/panel/content.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import panelContext from './context';
+
+const Form = styled.form`
```

Comment:
Looks like this would cause a form inside form due to `InspectorForm`

Reply:
Hm, what is `InspectorForm` doing? @spacedmonkey added that, right? Also, wasn't there also two forms inside each other before?

Reply:
I suppose it's used for `evt.target.form.dispatchEvent( new window.Event( 'submit' ) )` with `onBlur` for inputs.

Doesn't seem to have form inside form on `develop-stories`.

Reply:
> I suppose it's used for `evt.target.form.dispatchEvent( new window.Event( 'submit' ) )` with `onBlur` for inputs.

The reason I added for form wrappers was because of the submits here yes.

Reply:
Note: Now I can see the form in form errors on `develop-stories`, too. Mystery. Not sure if anything changed meanwhile.

Reply:
I've now removed the `InspectorForm` as I don't think it's being used. Each individual panel both before and now has their own form with an `onSubmit`, that handles those dispatches. The inspector form had no `onSubmit`, so any submit dispatched to it would actually just reload the entire page.

Everything still works in the panels with the inspector form removed from the inspector content as far as I can see.